src/Command_NICK.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Command.hpp"
+
+bool isSpecial(const char &chr) {
+	std::set<char> specialChars;
+	specialChars.insert('[');
+	specialChars.insert(']');
+	specialChars.insert('\\');
+	specialChars.insert('^');
+	specialChars.insert('_');
+	specialChars.insert('`');
+	specialChars.insert('{');
+	specialChars.insert('|');
+	specialChars.insert('}');
+	if (specialChars.find(chr) != specialChars.end())
+		return true;
+	return false;
```

Comment:
```suggestion
	return c != '[' && c != ']' && c != '\\' && c != '^' && c != '_' &&
		   c != '`' && c != '{' && c != '|' && c != '}';
```
単純にこのようなコードではない理由はありますか？

Reply:
元々は以下のような形で初期化できると思っていたのですが、この初期化の仕方はC++11以降でしか使用できず、今の形になりました。提供されたコードの方が見やすく理解しやすいと思うので取り入れます。
```
std::set<char> specialChars {'[',']','\\','|'....};
```